### DISCUSSION TITLE
Welcome to Tachidesk-Sorayomi Discussions!

### DISCUSSION THREAD

**Discussion by DattatreyaReddy** [Suwayomi/Tachidesk-Flutter#66] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Cipher-club** [comment]
how can i run this on a 32 bit windows?

**Reply by DattatreyaReddy** [reply]
32 bit of? (Android, IOS, Desktop)

**Comment by atmuio** [comment]
Hi, just installed whit homebrew, but when I start it he can't reach the server

**Reply by DattatreyaReddy** [reply]
you need to install Tachidesk server

**Reply by DattatreyaReddy** [reply]
https://github.com/Suwayomi/Suwayomi-Server

**Reply by atmuio** [reply]
my bad, thanks

**Comment by neo79747** [comment]
Good sir, my phone broke so i got the tachidesk but am finding it confusing to add extensions and stuff, if you can help me out with that please

**Reply by DattatreyaReddy** [reply]
please refer this https://github.com/Suwayomi/Suwayomi-Server/wiki/Getting-Extensions

as for extenison repository please search for it on internet

**Reply by DattatreyaReddy** [reply]
for now, there is no way to add extensions from Sorayomi

**Comment by bleacher333** [comment]
Some questions here since I'm planning on using the Suwayomi Server and has no experience using these apps before:

1. Is there some way to transfer the data from one computer to another? (i.e If I save some doujins and mangas on a PC, can I sync it or transfer that list to another PC?)
2. Is there any ways to import my fav list from nhentai to the saved library on Suwayomi? Or do I have to do it manually one by one?
3. If a work get deleted / purged or the website get taken down, would the saved manga still show up on the saved list in Suwayomi?

**Reply by DattatreyaReddy** [reply]
1. we have backup support (You can backup your data and restore it)
2. you have to do it manually
3. if you don't download it then you cant access them after website gets taken down (if you download them then you can view them without issues)

**Reply by bleacher333** [reply]
For no.3: Assuming I don't download them, would I still be able to see the name of the sites/works that was taken down/purged? Or it's gone as well and I won't be able to know what was deleted?
